Neos.Workspace.Ui/Classes/ViewModel/ChangeItem.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Neos.Workspace.Ui package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\Workspace\Ui\ViewModel;
+
+use Neos\Flow\Annotations as Flow;
+
+#[Flow\Proxy(false)]
+final readonly class ChangeItem
+{
+    public function __construct(
+        public string                  $serializedNodeAddress,
+        public bool                    $hidden,
+        public bool                    $isRemoved,
+        public bool                    $isNew,
+        public bool                    $isMoved,
+        public array                   $dimensions,
+        public ?string                 $lastModificationDateTime,
+        public ?string                 $createdDateTime,
+        public string                  $label,
+        public string                  $icon,
+        public ContentChangeItems      $contentChanges,
```

Comment:
Nitpick, but we don't really do these aligned arguments any more, because they don't really increase readability and lead to larger diffs when a new param is introduced that exceeds the previous max length
```suggestion
        public string $serializedNodeAddress,
        public bool $hidden,
        public bool $isRemoved,
        public bool $isNew,
        public bool $isMoved,
        public array $dimensions,
        public ?string $lastModificationDateTime,
        public ?string $createdDateTime,
        public string $label,
        public string $icon,
        public ContentChangeItems $contentChanges,
```